frontend/src/APIClients/CommonAPIClient.ts

```diff
@@ -84,6 +85,46 @@ const countLogRecords = async ({
   }
 };
 
+const getUsers = async (
+  pageNumber: number,
+  resultsPerPage: number,
+): Promise<GetUsersResponse> => {
+  try {
+    const bearerToken = `Bearer ${getLocalStorageObjProperty(
+      AUTHENTICATED_USER_KEY,
+      "accessToken",
+    )}`;
+    const { data } = await baseAPIClient.get<GetUsersResponse>(`/users`, {
+      params: {
+        pageNumber,
```

Comment:
Can we move these into a new file called `UserAPIClient.ts` - I think going forward `CommonAPIClient` would become too cluttered, so I wanna push for some architecture revision in terms of where we're placing client API utils. Specifically, I was thinking of something like this (similar to how we structure our types):

![Screen Shot 2023-07-12 at 11 57 07 PM](https://github.com/uwblueprint/supportive-housing/assets/53386475/fe8ade61-3ebf-4f77-a0ee-df9f9caacec6)

Reply:
Will be addressed in either of those PRs as per Slack convo